src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -250,12 +252,14 @@ const getDefaultIndex = (defaultOption: string | number, options: string[]) => {
 
 export const buildParameters = (
   parameters: ElementPluginParameters,
-  parameterLoadingValues: any
+  parameterLoadingValues: any,
+  internalParameters: string[]
 ) => {
   const simpleParameters: any = {}
   const parameterExpressions: any = {}
   for (const [key, value] of Object.entries(parameters)) {
-    if (value instanceof Object && !Array.isArray(value) && 'operator' in value) {
+    if (internalParameters.includes(key)) simpleParameters[key] = value
+    else if (value instanceof Object && !Array.isArray(value)) {
```

Comment:
I wonder what is this new `internalParameters` doing here. Is it worth adding a quick comment to explain this?

Reply:
Yeah, this is the problem Andrei and I were discussing [here](https://github.com/openmsupply/application-manager-web-app/pull/744), and I made an issue for it: #745.

What I've done here implements my suggested solution, but I'm not 100% sure that'll be what we go with -- I'll wait till @andreievg weighs in, or we can discuss it IRL tomorrow.